lib/sdf/link.rb

```diff
@@ -18,5 +18,39 @@ def each_sensor
         def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
+
+        # The link's inertial.
+        #
+        # @return Inertial struct
+        Inertial = Struct.new(:pose, :mass, :inertia)
+        def inertial
+            mass = read_float_child_element(xml, "mass", 1)
+            ixx = read_float_child_element(xml, "ixx", 1)
+            iyy = read_float_child_element(xml, "iyy", 1)
+            izz = read_float_child_element(xml, "izz", 1)
+            ixy = read_float_child_element(xml, "ixy", 0)
+            ixz = read_float_child_element(xml, "ixz", 0)
+            iyz = read_float_child_element(xml, "iyz", 0)
+            pose = Eigen::Isometry3.new
+             if inertial_elements = xml.elements["inertial"]
```

Comment:
Pure stylistic question (the code looks good). It seems that the if block is indented with one extra space.